src/jetson/AES.py

```diff
@@ -18,7 +18,10 @@ def __init__(self):
         self.salt = os.urandom(16) #Salt variable (Generates a random byte string)
         self.key = PBKDF2("passphrase", self.salt).read(16) #Creates key using KDF scheme
 
-    def encrypt(self, coordinates, image):
+    def encrypt(self,
+                coordinates: List[Tuple[int]],
+                image: 'numpy.ndarray[numpy.ndarray[numpy.ndarray[numpy.uint8]]]'):
```

Comment:
perhaps we could make this a [type alias](https://www.python.org/dev/peps/pep-0613/) so its easier to read and understand